### PR TITLE
Improve contribution guidelines

### PR DESCRIPTION
**Pull request by florianschmidt1994**
**What changes does this PR introduce**
- Add a CONTRIBUTING.md
- Add a SECURITY.md
- Add a MAINTAINERS.md
- Update README to link to CONTRIBUTING.md 

**Checklist**
- [x] `yarn build` passes
- [x] `yarn lint` does not show any errors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Contributing to QuarantäneHelden
+We really appreciate your help! We want to make contributing to this project as easy and transparent as possible, whether its:
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
We really appreciate your help! We want to make contributing to this project as easy and transparent as possible, whether it's:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ - Reporting bugs
+ - Proposing new features
+ - Submitting bug fixes
+ - Implementing features
+ 
+## We Develop with GitHub
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
## We develop with GitHub
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+We use GitHub issues to discuss bugs, features, improvements and meta-questions (such as this document). All contributions
+are submitted with pull requests against master.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
We use GitHub issues to discuss bugs, features, improvements and meta-questions (such as this document). All contributions are submitted with pull requests against **master**.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ - Submit your changes as a pull request against master
+
+## When opening a pull request, please:
+1. Ensure that you've discussed your change in a corresponding issue and you've been assigned to this issue by one of 
+the maintainers
+2. Ensure that any new code is covered by tests. Check out the `/cypress` directory for examples
+3. Ensure that your pull request does not introduce any new linting errors. You can run `yarn lint` in your local dev
+environment to verify this
+4. Only open a pull request if you consider it to be finished. Do not open "work-in-progress" pull requests, but 
+discuss your changes in an issue instead.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
4. Only open a pull request if you consider your code to be finished. Do not open "work-in-progress" pull requests, but 
discuss your changes in an issue instead
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+5. Make sure that your pull request is based against master.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
5. Make sure that your pull request is based against master
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Pull request review
+1. Request a review from one or multiple maintainers
+2. One of the maintainers will accept the invite for review, he will be your main reviewer for this pull request.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
2. One of the maintainers will accept the invite for review and will be your main reviewer for this pull request
```
Prevent gender

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+3. Your reviewer will ask for changes, if necessary. If no more changes seem necessary, your reviewer will accept your changes.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
3. Your reviewer will ask for changes, if necessary. If no more changes seem necessary, your reviewer will accept your changes
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+4. After accepting your changes, your reviewer will ask for a second pair of eyes from a second maintainer.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
4. After accepting your changes, your reviewer will ask for a second pair of eyes from a second maintainer
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+5. After both maintainers accept your changes, they will be squashed and merged into master.
```
Comment (Maurice22) [inline CONTRIBUTING.md]:
```suggestion
5. After both maintainers accept your changes, they will be squashed and merged into master
```

File: MAINTAINERS.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+**Janik**
+ - GitHub [@jagaSto](https://github.com/jagaSto)
+ 
+**Keno**
+- GitHub [@kenodressel](https://github.com/kenodressel) 
+- Mail [quarantaenehelden@keno.digital](mailto:quarantaenehelden@keno.digital)
+
+**Maurice**
+- GitHub [@Maurice22](https://github.com/Maurice22)
+
```
Comment (Maurice22) [inline MAINTAINERS.md]:
```suggestion
**Maurice**
- GitHub [@Maurice22](https://github.com/Maurice22)
- Mail [m.ackel@icloud.com](mailto:m.ackel@icloud.com)
```

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,6 @@
+# Security
+We greatly appreciate everybody who finds and discloses security vulnerabilities! If you want to disclose a a security
```
Comment (Maurice22) [inline SECURITY.md]:
```suggestion
We greatly appreciate everybody who finds and discloses security vulnerabilities! If you want to disclose a security
```

File: MAINTAINERS.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+**Keno**
+- GitHub [@kenodressel](https://github.com/kenodressel) 
+- Mail [quarantaenehelden@keno.digital](mailto:quarantaenehelden@keno.digital)
+
+**Maurice**
+- GitHub [@Maurice22](https://github.com/Maurice22)
+- Mail [m.ackel@icloud.com](mailto:m.ackel@icloud.com)
+**Tom**
+- GitHub [@tgraupne](https://github.com/tgraupne)
+- Mail [super-quarantaeneheld@tomgraupner.de](mailto:super-quarantaeneheld@tomgraupner.de)
```
Comment (tgraupne) [inline MAINTAINERS.md]:
```suggestion
**Maurice**
- GitHub [@Maurice22](https://github.com/Maurice22)
- Mail [m.ackel@icloud.com](mailto:m.ackel@icloud.com)

**Tom**
- GitHub [@tgraupne](https://github.com/tgraupne)
- Mail [super-quarantaeneheld@tomgraupner.de](mailto:super-quarantaeneheld@tomgraupner.de)
```

**Pull request merged**